cmd/provider_cmd_harvester.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/GoogleCloudPlatform/terraformer/providers/harvester"
+	"github.com/GoogleCloudPlatform/terraformer/terraformutils"
+)
+
+func newCmdHarvesterImporter(options ImportOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "harvester",
+		Short: "Import current state to Terraform configuration from Harvester",
+		Long:  "Import current state to Terraform configuration from Harvester",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			provider := newHarvesterProvider()
+			err := Import(provider, options, []string{})
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+	cmd.AddCommand(listCmd(newHarvesterProvider()))
+	baseProviderFlags(cmd.PersistentFlags(), &options, "image,virtualmachine", "ssh-user=ubuntu")
```

Comment:
prefer not to default to `ubuntu`, we can just leave it as empty

Reply:
This is `sampleFilters`, but it is still wrong, I changed it to `virtualmachine=id1:id2:id3`

Reply:
fixed.
eg:  only import image default/image-6skr8 and harvester-public/image-9lspw
```bash
terraformer-harvester import harvester -r image -f image=default/image-6skr8:harvester-public/image-9lspw
```